### ISSUE TITLE
The Travis CI access fail due of permission

### ISSUE THREAD

**Issue by dalisoft** [tweenjs/es6-tween#24]
@sole I can't access to settings. Or i should leave the app? If yes, i will leave :(

**Comment by dalisoft** [comment]
Sorry @sole. It's maybe mistake of [npmjs](https://github.com/npm/npm/issues/17195) or [travis], i not sure

**Comment by sole** [comment]
Sorry @dalisoft it is my fault! I revoked a token and forgot to update it on this repository. I will do it and update you here

**Comment by sole** [comment]
Is it working now? it looks like it should be?

**Comment by dalisoft** [comment]
@sole My tokens will been not a one, i am curius because nobody can't access to my profile and can't revoke my token without my permission, right? There it's not your fault. My tokens was used for 2 repos, not a one. Anyway i am already released release with my fork. I test it later

**Comment by dalisoft** [comment]
@sole I am still can't access to Travis CI

**Comment by sole** [comment]
I don't understand what you're trying to do - can you explain exactly what you're trying to do and where?

**Comment by dalisoft** [comment]
I cant set my token for es6 tween on travis to release new version

**Comment by sole** [comment]
Right, I don't really know what is actually happening. Looking at the travis output I can only see

```
> es6-tween@1.11.3 semantic-release /home/travis/build/tweenjs/es6-tween
> semantic-release pre && npm publish && semantic-release post --owner=sole --user=dalisoft
[Travis Deploy Once]: Success at attempt 1. All 1 jobs passed.
semantic-release ERR! pre Failed to determine new version.
semantic-release ERR! pre ENOCHANGE There are no relevant changes, so no new version is released.
```

and I don't know why is it failing to determine a new version.

I'll tell you what I've looked at, and let's see if together we can figure out what's wrong:

* In travis settings, under the tween.js/es6-tweenjs settings there are GH and NPM tokens

My first suspect is here: the NPM token belongs to the user [tweenjsadmin](https://www.npmjs.com/~tweenjsadmin) that I created so that publishes will not be done by me, but by that account.

But your semantic release configuration tries to publish as owner = sole using a token for tweenjsadmin, so that's not going to work. Maybe if you change the config to use the `tweenjsadmin` it will work?

**Comment by sole** [comment]
We also had [a similar issue](https://github.com/tweenjs/tween.js/issues/312#issuecomment-274893948) a while ago with `semantic-release` which couldn't determine the current version

**Comment by dalisoft** [comment]
Yes, only one account can be publish with Travis + semantic-release

**Comment by dalisoft** [comment]
```bash
semantic-release ERR! pre Failed to determine new version.
semantic-release ERR! pre ENOCHANGE There are no relevant changes, so no new version is released.
```.

Try trigger the version by comment on the commit
```
fix(travis): trigger travis, may fix the issue
```

**Comment by dalisoft** [comment]
@sole semantic-release fixes in `v2.2.1`, i did some config changes and improvements for performance and repo. I think you like it
